ShiftIt/ShiftIt-defaults.plist

```diff
@@ -85,14 +85,16 @@
 	<key>zoomKeyCode</key>
 	<integer>6</integer>
 	<key>zoomModifiers</key>
-    <integer>1835008</integer>
-   	<key>nextscreenKeyCode</key>
-   	<integer>45</integer>
-   	<key>nextscreenModifiers</key>
+	<integer>1835008</integer>
```

Comment:
Is this an intentional whitespace change?

Reply:
More or less.  I used Xcode's plist editor, and it automatically made the whitespace changes; but had I edited it manually I would have likely done the same thing to make the file more consistent.